server/src/runtime.ts

```diff
@@ -119,11 +119,16 @@ export class ScryptedRuntime extends PluginHttp<HttpPluginData> {
                 const spawn = require('node-pty-prebuilt-multiarch').spawn as typeof ptySpawn;
                 const cp = spawn(process.env.SHELL, [], {
                 });
-                cp.onData(data => connection.send(JSON.stringify({data})));
+                cp.onData(data => connection.send(data));
                 connection.on('message', message => {
+                    if (Buffer.isBuffer(message)) {
+                        cp.write(message.toString());
```

Comment:
Why toString here, can write the buffer?

Reply:
According to this, they are expecting a string https://github.com/microsoft/node-pty/blob/main/typings/node-pty.d.ts#L168

Reply:
Huh, weird. Looks good then